### PR TITLE
Modify the scripts to release from master a new version 7.0.0

### PR DESCRIPTION
**Pull request by anechifor**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -88,19 +94,13 @@ jobs:
     - name: "Release"
       stage: release
       name: "Push to Nexus"
-      if: commit_message ~= /\[trigger release\]/ AND branch ~= /^(master|SP\/.+|HF\/.+)$/
+      if: fork=false AND commit_message ~= /\[trigger release\]/ AND branch ~= /^(master|SP\/.+|HF\/.+)$/
```
Comment (cristina-sirbu) [inline .travis.yml:97]: When we trigger a release we are only pushing to Nexus?

File: _ci/build.sh [hunk 2]
```diff
@@ -8,7 +8,7 @@ pushd "$(dirname "${BASH_SOURCE[0]}")/../"
 source _ci/init_tag.sh
 
 # Change tag if you are on a branch
-if [ ! -z "$TRAVIS_BRANCH" -a "$TRAVIS_BRANCH" != "master" ]; then
```
Comment (cristina-sirbu) [inline _ci/build.sh:11]: Why do we add TAG_NAME on master image?

File: _ci/init_tag.sh [hunk 3]
```diff
@@ -7,13 +7,19 @@ echo "The branch is ${TRAVIS_BRANCH}"
 echo "Project version: ${PROJECT_VERSION}"
 
 if [ "${TRAVIS_BRANCH}" = "master" ]; then
-  TAG_NAME="latest"
+  TAG_NAME=$latest
```
Comment (cristina-sirbu) [inline _ci/init_tag.sh:10]: Where is latest variable defined?

File: .travis.yml (hunk 1 above)
Comment (anechifor) [reply .travis.yml:97]: yes this is the approach, acs use the share.war.

File: _ci/build.sh (hunk 2 above)
Comment (anechifor) [reply _ci/build.sh:11]: I use the same variable name on build.sh. Only for consistency.

File: .travis.yml (hunk 1 above)
Comment (cristina-sirbu) [reply .travis.yml:97]: So we will not create a docker image with this war? At least on quay?

**Pull request merged**